packages/slider/src/Slider.js

```diff
@@ -79,11 +80,16 @@ export default class Slider extends Component {
     /**
      * Value of the field
      */
-    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
+    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    /**
+     * Type of the slider
+     */
+    type: PropTypes.oneOf(AVAILABLE_SLIDER_TYPES)
```

Comment:
Instead of `type` we should use `variant` as we are moving towards using that instead of `type` especially since `type` is an HTML attribute.